manifests/4.9/kube-descheduler-operator.crd.yaml

```diff
@@ -11,167 +11,144 @@ spec:
-                format: int32
-              logLevel:
-                description: "logLevel is an intent based logging for an overall component.
-                  \ It does not give fine grained control, but it is a simple way
-                  to manage coarse grained logging choices that operators have to
-                  interpret for their operands. \n Valid values are: \"Normal\", \"Debug\",
-                  \"Trace\", \"TraceAll\". Defaults to \"Normal\"."
-                type: string
-                default: Normal
-                enum:
-                - ""
-                - Normal
-                - Debug
-                - Trace
-                - TraceAll
-              managementState:
-                description: managementState indicates whether and how the operator
-                  should manage the component
-                type: string
-                pattern: ^(Managed|Unmanaged|Force|Removed)$
-              observedConfig:
-                description: observedConfig holds a sparse config that controller
-                  has observed from the cluster state.  It exists in spec because
-                  it is an input to the level for the operator
-                type: object
-                nullable: true
-                x-kubernetes-preserve-unknown-fields: true
-              operatorLogLevel:
-                description: "operatorLogLevel is an intent based logging for the
-                  operator itself.  It does not give fine grained control, but it
-                  is a simple way to manage coarse grained logging choices that operators
-                  have to interpret for themselves. \n Valid values are: \"Normal\",
-                  \"Debug\", \"Trace\", \"TraceAll\". Defaults to \"Normal\"."
-                type: string
-                default: Normal
-                enum:
-                - ""
-                - Normal
-                - Debug
-                - Trace
-                - TraceAll
-              profiles:
-                description: Profiles sets which descheduler strategy profiles are
-                  enabled
-                type: array
-                items:
-                  description: DeschedulerProfile allows configuring the enabled strategy
-                    profiles for the descheduler it allows multiple profiles to be
-                    enabled at once, which will have cumulative effects on the cluster.
+    - name: v1
+      schema:
+        openAPIV3Schema:
+          description: KubeDescheduler is the Schema for the deschedulers API
+          type: object
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
@soltysh this seems to be undoing some of the changes from your updates in https://github.com/openshift/cluster-kube-descheduler-operator/pull/205/commits/53cadec8cf411975c075f8258c4da595b5bf75f6

This was run with just `make update-codegen-crds` (when I could get it to run and not segfault)... how did you make your changes there?

Reply:
The entire block is indented to the right a bit. Any way to move it to the original column?

Reply:
> when I could get it to run and not segfault
Can you more elaborate on how did you get around the segfault?

Reply:
I just kept re-running it until it passed. I think there is some sort of race in our version of the generator